compiler/test/fail_compilation/fail10254.d

```diff
@@ -2,14 +2,14 @@
 TEST_OUTPUT:
 ---
 fail_compilation/fail10254.d(20): Error: `pure` function `fail10254.foo` cannot call impure constructor `fail10254.C.this`
-fail_compilation/fail10254.d(20): Error: `@safe` function `fail10254.foo` cannot call `@system` constructor `fail10254.C.this`
 fail_compilation/fail10254.d(15):        `fail10254.C.this` is declared here
 fail_compilation/fail10254.d(21): Error: `pure` function `fail10254.foo` cannot call impure constructor `fail10254.S.this`
-fail_compilation/fail10254.d(21): Error: `@safe` function `fail10254.foo` cannot call `@system` constructor `fail10254.S.this`
 fail_compilation/fail10254.d(16):        `fail10254.S.this` is declared here
```

Comment:
Isn't it better to diagnose both `pure` and `@safe` violations at the same time?